dev/tasks/linux-packages/apache-arrow/yum/arrow.spec.in

```diff
@@ -834,6 +834,9 @@ Documentation for Apache Parquet GLib.
 %{_datadir}/gtk-doc/html/parquet-glib/
 
 %changelog
+* Mon Jan 24 2022 Krisztián Szűcs <szucs.krisztian@gmail.com> - 7.0.0-1
```

Comment:
@kou seems like we also need update the linux packaging changelogs in https://github.com/apache/arrow/blob/master/dev/release/post-12-bump-versions.sh

Reply:
Ah, you're right. I'll do it.

BTW, we also need `post-XX-changelog.sh` that runs `archery release changelog add $version`. See also: ARROW-13460

Reply:
Added a ticket https://issues.apache.org/jira/browse/ARROW-15449